Storage/tests/System/ManageBucketsTest.php

```diff
@@ -464,4 +479,18 @@ public function autoclassConfigs()
             [['enabled' => true, 'terminalStorageClass' => 'ARCHIVE']],
         ];
     }
+
+    public function hnsConfigs()
+    {
+        return [
+            [[], false],
+            [['hierarchicalNamespace' => ['enabled' => false,]], false],
+            // TODO: Test cases fails because of no uniformBucketLevelAccess
+            // [['hierarchicalNamespace' => ['enabled' => true,]], false],
```

Comment:
It might be unnecessary to test for scenario that create Bucket with HNS enabled fails without `uniformBucketLevelAccess` enabled.

Reply:
@frankyn is this required?

Reply:
Recommend not testing this case since HNS always requires UBLA enabled.